tck/scala-tck/src/main/scala/akkaserverless/tck/model/TckService.scala

```diff
@@ -20,6 +20,7 @@ import scala.concurrent.duration._
 import com.akkaserverless.scalasdk.PassivationStrategy
 import com.akkaserverless.scalasdk.eventsourcedentity.EventSourcedEntityOptions
 import com.akkaserverless.scalasdk.valueentity.ValueEntityOptions
+import com.akkaserverless.tck.model.Main
```

Comment:
shouldn't this whole file be moved to `com.akkaserverless.tck.model`?

Reply:
yep, better to put them all together